src/PaymentGateways/Gateways/Stripe/Migrations/AddStatementDescriptorToStripeAccounts.php

```diff
@@ -27,7 +30,14 @@ public function run()
             foreach ($allStripeAccount as $index => $stripAccount) {
                 if (!isset($stripAccount['statement_descriptor'])) {
                     $statementDescriptor = trim($statementDescriptor);
-                    $this->validateStatementDescriptor($statementDescriptor);
+
+                    try {
+                        $this->validateStatementDescriptor($statementDescriptor);
```

Comment:
The method `validateStatementDescription` throws, specifically, a `InvalidArgumentException`, but we are catching here in a generic `Exception`. We should probably, instead, create a custom exception that extends `InvalidArgumentException` and then catch _that_ exception.